src/components/challenge/component.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styles from "./_.module.css";
+import {Link} from "react-router-dom";
+import CNST from "../../constants";
+import {getRandomImage} from "../../helpers/getRandomImage.js";
+
+import userImg from "../../images/user.png";
+import dotsIcon from "../../images/dots.svg";
+import voteIcon from "../../images/vote.svg";
+
+export const Challenge = ({isOpen = true, data}) => {
+  const {name, description, username} = data;
+  const challengeImg = getRandomImage();
+
+  return (
+    <section className={styles.challenge}>
+      <header className={styles.header}>
+        <img src={userImg} alt="User" className={styles.userImg} />
+        <span className={styles.username}>{username}</span>
+        <img src={dotsIcon} alt="More" />
+      </header>
+      <img src={challengeImg} alt="Challenge" className={styles.challengeImg} />
+      <Link to={CNST.ROUTES.HOME}>
+        <div className={styles.footer}>
+          <div className={styles.votesContainer}>
+            <span className={styles.id}>#100245</span>
+            {!isOpen && (
+              <>
+                <img src={userImg} alt="Winner" className={styles.winnerImg} />
+                <span className={styles.winnerName}>Loulou won the challenge</span>
```

Comment:
we can add 3 dots for the name it is a long name
![image](https://user-images.githubusercontent.com/82516098/125385789-dd7c0900-e3a3-11eb-8262-855bcb9a5c68.png)